recipes/janome/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "janome" %}
+{% set version = "0.4.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/Janome-{{ version }}.tar.gz
+  sha256: 6c2c38d894014d57cb3151265c11146506ead3b3bc290898adc33711711612de
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+
+test:
+  imports:
+    - janome
+    - janome.sysdic
+
+about:
+  home: https://github.com/mocobeta/janome
+  summary: Japanese morphological analysis engine.
+  license: Apache-2.0
+  license_file: LICENSE.txt
```

Comment:
Go ahead and add `NOTICE.txt` and then here you can use a list:
```yaml
license_file:
  - LICENSE.txt
  - NOTICE.txt
```

Reply:
Thanks for the quick review. Should be fixed now.